cli/js/web/streams/internals.ts

```diff
@@ -311,85 +311,78 @@ export function isDetachedBuffer(value: object): boolean {
 }
 
 function isFiniteNonNegativeNumber(v: unknown): v is number {
-  return Number.isFinite(v) && (v as number) >= 0;
+  return Number.isFinite(v as number) && (v as number) >= 0;
```

Comment:
Why add an unnecessary cast?

Reply:
> Why add an unnecessary cast?

TS2345: Argument of type 'unknown' is not assignable to parameter of type 'number'.

Reply:
I don't understand. My intellisense says that `Number.isFinite()` accepts `unknown`. It's been compiling successfully already, right?

Reply:
> I don't understand. My intellisense says that `Number.isFinite()` accepts `unknown`. It's been compiling successfully already, right?

I'm using WebStorm, not VS Code

<img width="1440" alt="Screenshot 2020-07-06 at 18 34 34" src="https://user-images.githubusercontent.com/62983943/86611466-5c7f7a00-bfb7-11ea-91ae-b23923afe973.png">

Reply:
This sounds like an issue with your plugin. That's the one which doesn't match the library Deno is using. You should raise an issue there.

-1 for arbitrarily accommodating incompatible type definitions.

Reply:
Well or inaccurate ones that are located in just an editor.  If it doesn't break on compile, it isn't valid, irrespective of what the editor says.